javalib/src/main/scala/java/lang/UnixProcess.scala

```diff
@@ -6,10 +6,17 @@ import java.util.concurrent.TimeUnit
 import java.util.ScalaOps._
 import scala.scalanative.unsigned._
 import scala.scalanative.unsafe._
-import scala.scalanative.libc.{errno => err, signal => sig, _}
-import sig._
+import scala.scalanative.libc.{errno => err}
```

Comment:
This is a second generation, picky mode, observation, not meant
to hold up the merge of the PR.

The distinction between the errno intended by 'err` and `e`
is pretty fine.  Given the path that got us to having two subtlety 
different errnos, I suggest making it easier for future readers
to determine, at the point of reading,  which one is meant. 

Something like:
'```
import scala.scalanative.libc.{errno => libcErrno}
import scala.scalanative.posix.{
  fcntl,
  pthread,
  signal,
  sys,
  time,
  unistd,
  errno => posixErrno
}
```
Eric, If you agree with the concept but do not have time/energy to make
the fix in this PR.  I can alway submit a followup PR.  Let he/she who proposes, dispose (An old Digital Equipment Corporation saying/principal)
to keep work on a given item from expanding without end and consuming
the person doing the original work).

Reply:
I am just removing the `signal` import and not changing the other anything else. I think that is the correct action.

Reply:
I think working the errno is out of scope. Maybe they should be `cerr` and `perr` but without the extra imports to bring more things into scope.

Reply:
Understood & agreed; minimal changes and those for only the current topic.

I will add UnixProcess.scala to the list of places I hope to never visit....